ops/prod/api.tf

```diff
@@ -4,7 +4,7 @@ module "simple_report_api" {
   env    = local.env
 
   instance_count = 4
-  instance_size  = "P2v2"
+  instance_size  = "P3v3"
```

Comment:
The PR description says this should be `P2v3` - which value is correct? `P3v3` is a pretty big cost jump.

Reply:
Also, even if we just jump from v2 to v3 and keep the instance size the same, it's a ~50% cost increase.

Reply:
Whoops, sorry I forgot to update the PR description. I chatted with @zdeveloper about this this morning - apparently Tim Best approved this upgrade specifically. I'm not sure if we really need 32 GB of memory, but I think at least the 16 provided by P2v3 is needed given our increase in traffic

Reply:
We're all good then if that's the case.

Reply:
Yeah, after chatting with Tim Best, he said we can go to P3v3 and observe the audit log, and if its an over kill, its always easy to scale down to P2v3